src/java/org/apache/cassandra/service/ClientState.java

```diff
@@ -100,8 +100,8 @@
             }
             catch (Exception e)
             {
-                JVMStabilityInspector.inspectThrowable(e);
-                logger.info("Cannot use class {} as query handler ({}), ignoring by defaulting on normal query handling", customHandlerClass, e.getMessage());
+                logger.error("Cannot use class {} as query handler ({})", customHandlerClass, e.getMessage());
```

Comment:
In our case, the exception thrown by the QueryHandler is a cassandra ConfigurationException, which has nested exception that gives valuable information to the user. 
In that sense, I'd recommend to pass the whole exception to the logger rather than just the message, like:
`
logger.error("Cannot use class {} as query handler ({})", customHandlerClass, e);
`